elastic/apm-server
# Support custom response headers for RUM endpoints

## Checklist


I have considered changes for:

## How to test these changes

Configure some headers in apm-server.rum.response_headers, send a request to any/all rum endpoints, and observe those headers in the response. Should work for both strings and lists

## Related issues

Closes #3696
Docs issue #3821
